Cartfile.private

```diff
@@ -1,3 +1,4 @@
 github "erikdoe/ocmock" ~> 3.3
 github "facebook/ios-snapshot-test-case" ~> 2.1
 github "jspahrsummers/xcconfigs" ~> 0.9
+github "facebook/yoga" ~> 1.2
```

Comment:
I'm not sure this dependency can go in this file. Technically Yoga is a public dependency. Anybody depending on ComponentKit will also need to get Yoga as well.